README.md

```diff
@@ -226,7 +226,8 @@ Describes the current generation of the `cellular` connection. It is an enum wit
 | `2g`      | Currently connected to a 2G cellular network. Includes CDMA, EDGE, GPRS, and IDEN type connections                |
 | `3g`      | Currently connected to a 3G cellular network. Includes EHRPD, EVDO, HSPA, HSUPA, HSDPA, and UTMS type connections |
 | `4g`      | Currently connected to a 4G cellular network. Includes HSPAP and LTE type connections                             |
-
+| `5g`      | Currently connected to a 5G cellular network. Includes NR type connections                                        |
```

Comment:
```suggestion
| `5g`      | Currently connected to a 5G cellular network. Includes NR and NRNSA type connections                                        |
```

Looks like there are two based on the ios code https://github.com/react-native-netinfo/react-native-netinfo/pull/436/files#diff-facfca1c228756e5807b5c125ee476d05edc9bad63f34479d55c8b262e6d5c51R65

What do you think?

Reply:
Sorry for the late reply. Yep, looks like there are 2 types on iOS side. 
So, do we need to reflect the difference, like NR for Android, NR and NRNSA for iOS

Reply:
Yes, I'd include them all - just following previous pattern

Reply:
Well, I don't find a similar patter to follow. Like for most sections, there is a platform column, and the only one that differentiate the platforms without the column is the NetInfoState one (https://github.com/react-native-netinfo/react-native-netinfo#netinfostate). 

So I changed it to "Includes NRNSA (iOS only) and NR type connections". Does this work?